src/coreclr/src/vm/loaderallocator.cpp

```diff
@@ -1222,6 +1226,10 @@ void LoaderAllocator::Init(BaseDomain *pDomain, BYTE *pExecutableHeapMemory)
         m_callCountingManager = new CallCountingManager();
     }
 #endif
+
+#ifdef FEATURE_ON_STACK_REPLACEMENT
+    m_onStackReplacementManager = new OnStackReplacementManager();
```

Comment:
It may be nice to allocate this lazily only once there are active patch points to manage.

Reply:
Done.